Objects/regions.c

```diff
@@ -1316,7 +1325,7 @@ static PyObject *add_to_region(PyObject *obj, Py_region_ptr_t region)
 
 int _Py_is_bridge_object(PyObject *op) {
     Py_region_ptr_t region = Py_REGION(op);
-    if (IS_LOCAL_REGION(region) || IS_IMMUTABLE_REGION(region)) {
+    if (IS_LOCAL_REGION(region) || IS_IMMUTABLE_REGION(region) || IS_COWN_REGION(region)) {
```

Comment:
Small (optional) suggestion:
```suggestion
    if (!HAS_METADATA(region)) {
```

Reply:
I think it is better to keep this code as is. The reason for op not being a bridge is because it is in the local region, is immutable or a cown -- not that it is an object that does not have metadata.